utils/fhirpath/Ballerina.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+org = "ballerinax"
+name = "health.fhir.r4utils.fhirpath"
+version = "1.0.3-snapshot"
```

Comment:
Let's not use snapshots

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerinax-health.fhir.r4/commit/08e57e7fae8edd666b262458106711c51a4fd471